tools/gulp/docs/index.js

```diff
@@ -63,6 +63,12 @@ function addTopLevelPages(kssSections) {
         sections: [],
         weight: 7
       },
+      {
```

Comment:
I think this was missing before because Utilities was the only top level section that had a "Overview" page (it was labeled "Utilities"). As a result the top level section and it's corresponding nav label was kind of built dynamically from the KSS (I think?). So basically I made the top level section explicitly defined here instead.